x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/source_settings.tsx

```diff
@@ -180,6 +220,57 @@ export const SourceSettings: React.FC = () => {
           </EuiFormRow>
         </ContentSection>
       )}
+      {showSyncControls && (
+        <ContentSection title={SYNC_MANAGEMENT_TITLE} description={SYNC_MANAGEMENT_DESCRIPTION}>
+          <EuiPanel hasShadow={false} hasBorder grow={false}>
+            <EuiFlexGroup>
+              <EuiFlexItem grow={false}>
+                <EuiSwitch
+                  checked={synchronizeChecked}
+                  onChange={(e) => setSynchronize(e.target.checked)}
+                  label={SYNC_MANAGEMENT_SYNCHRONIZE_LABEL}
+                  data-test-subj="SynchronizeToggle"
+                />
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiButton
+                  color="primary"
+                  onClick={submitSyncControls}
+                  data-test-subj="SaveSyncControlsButton"
+                >
+                  {SAVE_CHANGES_BUTTON}
```

Comment:
It appears that the "Save changes" button is always enabled, even if there are no changes. This could be confusing, as we have a pattern of keeping the buttons disabled when there are no changes. Was this discussed with PM/design at all? 

Also, I don't see a linked issue in this PR and have some questions about the design. 
![image](https://user-images.githubusercontent.com/1869731/129402832-34267786-915a-4015-8626-b82b38db0d97.png)

1. It seems odd to have a border around this. I understand why, but we don't use this pattern anywhere else that I can recall so just wanted to point it out
2. It seems weird that this is a full-width box, since the feature itself doesn't even take up half of the screen. Perhaps this would look better without being full width?
3. The button just looks off here. Perhaps we could either use a smaller button or put this under the text? 

Apologies if this has already been decided upon; I just thought it worth bringing up. Would love to get thoughts from @zombieFox and @bhuvanaurora

Reply:
Got it, I've updated the PR description with the associated [issue](https://github.com/elastic/workplace-search-team/issues/1495).

Reply:
Thanks. The design actually addresses just about everything I mentioned. Is there a reason we didn't go with that design?

![image](https://user-images.githubusercontent.com/1869731/129407536-48dc7d85-e2b1-4f9b-a768-bfb3f6752264.png)

Reply:
Ah, it's been too long since I've reference that mockup. No good reason it wasn't followed. I can try and clean that up in this PR.

Reply:
To simplify things, the new changes remove the border and move the button to the bottom so we're not trying to vertically align it:
<img width="849" alt="Screen Shot 2021-08-13 at 2 46 36 PM" src="https://user-images.githubusercontent.com/1614023/129411153-0862be6f-8f14-4f5d-94cb-c14dc8ce86b2.png">

@scottybollinger thoughts?

Reply:
I like this a lot better. What I don't like is there is no feedback for the user to know there are changes that have not been saved. As I mentioned above, we typically have a disabled button state that only becomes enabled when there are unsaved changes. Otherwise, how is a user to know if they have made any changes and whether they have been persisted or not. This may or may not be a legitimate concern. Will leave it up to Product to determine if this is needed and if so, as a part of this PR or not.

Reply:
Feel free to merge this, as implementing the dirty/unsaved logic will require the state logic refactor I was talking about earlier and might take us past FF.

Reply:
Makes sense 👍